src/Sylius/Bundle/ProductBundle/Form/Type/ProductType.php

```diff
@@ -34,6 +34,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('attributes', 'collection', array(
                 'required'     => false,
                 'type'         => 'sylius_product_attribute_value',
+                'prototype' => false,
```

Comment:
I think array should be whole aligned or not aligned... Did we agree that aligning "arrows" is cheesy or not? :smile: